instructure/canvas-ios
# Add assignment reminders UI

refs: MBL-17296
affects: Student
release note: none

test plan:
- Turn on assignment reminder experimental flag.
- Go to an assignment that has a due date set in the future.
- Check if add reminder section is visible.
- Add pre defined and custom reminders.
- Only UI, there's no business logic or notifications really scheduled.

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td><img src="https://github.com/instructure/canvas-ios/assets/72396990/e84abaf5-ee3a-4a62-9f5e-cb660ed1b55c" maxHeight=500></td>
<td><img src="https://github.com/instructure/canvas-ios/assets/72396990/08d46d13-2bfa-428a-aae7-4b325bbb713f" maxHeight=500></td>
</tr>
<tr>
<td><img src="https://github.com/instructure/canvas-ios/assets/72396990/4f4bb709-446b-4413-ac5e-79cb89ee17e8" maxHeight=500></td>
<td><img src="https://github.com/instructure/canvas-ios/assets/72396990/f1596ffe-be65-404f-b8e4-1ee7684e9c5c" maxHeight=500></td>
</tr>
</table>

## Checklist
